### DISCUSSION TITLE
How to build xmake with system libraries instead of third-party external libraries?

### DISCUSSION THREAD

**Discussion by topazus** [xmake-io/xmake#3726] [Q&A]
Since most of the Linux distributions have packaged some dependencies, like lua and lz4, so I wonder that how to build xmake against these system packages?

**Comment by waruqi** [comment]
see https://xmake.io/#/package/system_package

**Reply by topazus** [reply]
> see [xmake.io/#/package/system_package](https://xmake.io/#/package/system_package)

The link you gave seems to be about the usage of xmake. Maybe my previous description is not clear, I mean how to build xmake package with system packages.

build log: https://gist.githubusercontent.com/topazus/69dca44a4fc7c6d00869b3cbb2f06ad7/raw/9b73977253a96eceee949c76b8abb421a722e39a/build.log

**Reply by waruqi** [reply]
see https://xmake.io/#/guide/installation?id=source-compilation-and-installation
